src/amg/setup.py

```diff
@@ -33,7 +33,7 @@
 ]
 
 # TODO: Add any additional SDK dependencies here
-DEPENDENCIES = []
+DEPENDENCIES = ['azure-mgmt-dashboard >= 1.1.0']
```

Comment:
I noticed you moved `azure-mgmt-dashboard` from the `vendored_sdk` into `DEPENDENCIES`. Mind sharing the reasoning behind this change? It'd be helpful to understand the context.

Reply:
We were copying the entire SDK and storing it in the extension before. Now we decided to move to using the PyPI package instead of copying over all the files manually.

Reply:
This reduces a lot of lines in our project and also the next time our API spec changes we can simply bump the dependency version instead of copying over the entire SDK again